src/pages/overlay/components/AmbassadorList.tsx

```diff
@@ -34,6 +29,11 @@ function scrollListToAnimal(listElement: HTMLDivElement | null, name: string) {
     }
 }
 
+export interface AmbassadorListProps{
+    showAnimalList: boolean
+    chatChosenAmbassador?: string
+}
+
```

Comment:
remove new line in between AmbassadorListProps and AmbassadorList

Reply:
should be fixed